### ISSUE TITLE
Proposal: new repeat list mode: Per Element

### ISSUE THREAD

**Issue by hell2o** [JacquesLucke/animation_nodes#1372]
It would be useful to have an easy way to duplicate elements in a list such that:
{1,2,3}  becomes {1,1,2,2,3,3} (for amount = 2) or {1,1,1,2,2,2,3,3,3} (for amount = 3)
This would only make sense in the "amount" mode not the lengths modes.
It's possible I've missed something but for now it seems like the easiest way to do this is to create a subprogram for it, which seems quite complicated for such a simple operation.

**Comment by OmarEmaraDev** [comment]
I think this will be helpful indeed. We can add this feature.

**Comment by JacquesLucke** [comment]
Sounds good. 
What do you think about a `Repeat List Elements` node. It would have a `List` and `Amount` input. The `Amount` input should be vectorized.

```python
new_list = []
for element, amount in zip(old_list, amounts):
    new_list.extend([element] * amount)
```

**Comment by OmarEmaraDev** [comment]
Implemented in bc0b613. Thanks for the suggestion!
